modules/ROOT/pages/con_tasks-syntax.adoc

```diff
@@ -0,0 +1,74 @@
+[]
+= Tasks Syntax
+
+To define a configuration file for a `Task` resource, specify the
+following fields:
+
+* Required:
+  * `apiVersion` - Specifies the API version.
+  * `kind` - Specify the `Task` resource object
+  * `metadata` - Specifies data to uniquely identify the
+    `Task` resource object, for example a `name`.
+  * `spec` - Specifies the configuration information for
+    your `Task` resource object. `Task` steps must be defined through either of
+    the following fields:
+  ** link:https://github.com/tektoncd/pipeline/blob/master/docs/tasks.md#steps[`steps`] - Specifies one or more container images that you want
+      to run in your `Task`.
```

Comment:
@joaedwar: Here the link statement is correct and it renders correctly! :)
steps- Specifies--> steps- Specify